uamqp/client.py

```diff
@@ -1097,7 +1102,7 @@ def receive_messages_iter(self, on_message_received=None):
         :type on_message_received: callable[~uamqp.message.Message]
         """
         self._message_received_callback = on_message_received
-        self._received_messages = compat.queue.Queue()
+        self._received_messages = self._received_messages or compat.queue.Queue()
```

Comment:
Is this assignment statement still necessary? `self._received_messages` should never be `None` now right?